README.md

```diff
@@ -4,7 +4,7 @@
   
   <img src="./img/logo2.png" alt="logo" width="140"  height="auto" />
   <br/>
-
```

Comment:
### CONTRIBUTING SECTION

- [ ] Please, kindly add the link to the Github issues page of this project to make the users quickly navigate to the issue page.